espresso/core/java/androidx/test/espresso/ViewInteraction.java

```diff
@@ -333,6 +341,58 @@ public Void call() {
     return this;
   }
 
+  /**
+   * Captures an image of the underlying view into a {@link Bitmap}.
+   *
+   * <p>Essentially makes {@link ViewCapture#captureViewToImage(View)} available for users using
+   * espresso API.
+   *
+   * <p>This API is currently experimental and subject to change or removal.
+   */
+  @ExperimentalScreenshot
+  public Bitmap captureToImage() {
```

Comment:
Feels like an unnecessary API surface, does it need special treatment rather than `.perform(takeScreenshot())`?

Reply:
Thanks for the replies, makes sense.